Assignment/Program.cs

```diff
@@ -1,26 +1,27 @@
 ﻿using System.Text;
-for (; ; ) {
-   Console.Write ("Enter a number : ");
-   string input = Console.ReadLine ();
-   if (int.TryParse (input, out int num)) {
-      ToBinary (num);
-      ToHexaDecimal (num);
+using static System.Console;
+while (true) {
+   Write ("Enter a number : ");
+   if (int.TryParse (ReadLine (), out int num) && num >= 0) {
+      if (Convert (num, 2) == ToBinary (num) && Convert (num, 16) == ToHexa (num)) {

```

Comment:
(Convert (num, 2) and (Convert (num, 16) called twice.
Also the check is not needed.
Just print them maybe like Binary computation and Binary string conversion etc.